windows/FileUtils.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Threading.Tasks;
+using Windows.Storage;
+
+namespace CodePush.ReactNative
+{
+    class FileUtils
+    {
+        public async static Task MergeDirectories(StorageFolder source, StorageFolder target)
+        {
+            foreach (StorageFile sourceFile in await source.GetFilesAsync())
+            {
+                await sourceFile.CopyAndReplaceAsync(await target.CreateFileAsync(sourceFile.Name, CreationCollisionOption.OpenIfExists));
```

Comment:
yea, looks like you would have to use `CopyAsync(destinationFolder, newName, nameCollisionOption)` and pass in the existing name as the new name and replaceIfExisting for the nameCollisionOption